src/components/CartTotals.js

```diff
@@ -6,9 +6,7 @@ import { formatPrice } from '../utils/helpers';
 import { Link } from 'react-router-dom';
 
 const CartTotals = () => {
-  const { total_amount, shipping_fee } = useCartContext();
-  // TODO: should tax maybe be a percentage of the subtotal? I suppose this may be different in different regions of the world.
-  const taxRate = 0.15; // 15% tax rate
+  const { total_amount, shipping_fee, taxRate } = useCartContext();
```

Comment:
moved tax rate into the context component. This way we can easily use the same value both in `CartTotals` and `Orders` components